test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -246,7 +248,7 @@ public void AutoFitRecommendationTest()
             RunDetail<RegressionMetrics> bestRun = experimentResult.BestRun;
             Assert.True(experimentResult.RunDetails.Count() > 1);
             Assert.NotNull(bestRun.ValidationMetrics);
-            Assert.True(experimentResult.RunDetails.Max(i => i.ValidationMetrics.RSquared != 0));
+            Assert.True(experimentResult.RunDetails.Max(i => i?.ValidationMetrics?.RSquared) != 0);
```

Comment:
May want to check that R<sup>2</sup> is `>0.5` (or any other expected value). Being exactly `0.0` would be odd as the range for R<sup>2</sup> is `-Infinity` to `1.0`.
```suggestion
            Assert.True(experimentResult.RunDetails.Max(i => i?.ValidationMetrics?.RSquared) > 0.5);
```